### PR TITLE
OmniVehicle config changes require omnipod heat sinks

### LINKED ISSUE #2036: OmniVehicle config changes use regular heat sinks
Another minor refit issue:

When changing to an OmniVehicle config that requires more heat sinks, regular heat sinks will be used instead of OmniPodded ones.

To reproduce:

1. Add (GM) a Badger Tracked Transport Prime
2. Change to config D and note that it ordered a regular, non-OmniPodded heat sink
3. Chnage back to config Prime and note the heat sink returned to the warehouse is not OmniPodded

### PR DESCRIPTION
**Pull request by RexPearce**
Fixes #2036

### CODE REVIEW

**Review by Windchild292 — APPROVED** [review]

File: MekHQ/src/mekhq/campaign/parts/Refit.java [hunk 1]
```diff
@@ -902,6 +902,9 @@ public void calculate() {
             }
             if (oldHS != newHS) {
                 Part hsPart = heatSinkPart(newEntity); // only single HS allowed, so they have to be of the same type
+                if (isOmniRefit) {
+                    hsPart.setOmniPodded(true);
+                }
```
Comment (Windchild292) [inline MekHQ/src/mekhq/campaign/parts/Refit.java:907]: Couldn't we just set this to omniRefit and ignore the if statement?

Comment (RexPearce) [reply MekHQ/src/mekhq/campaign/parts/Refit.java:907]: Unless I'm missing something, the block starting a L904 is reached by any vehicle refit that changes the number of external (non-engine-free) heat sinks. (i.e. not just OmniVehicles)

Comment (Windchild292) [reply MekHQ/src/mekhq/campaign/parts/Refit.java:907]: Correct, but is the setOmniPodded to isOmniRefit cheaper than an if statement followed the setter?

Comment (RexPearce) [reply MekHQ/src/mekhq/campaign/parts/Refit.java:907]: Ah, I understand what you're saying  now - good suggestion.

**Pull request merged**